File: Java/bee 1008.java
import java.util.Scanner;

public class Main {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int num, horasTrabalhadas;
        num = sc.nextInt();
        horasTrabalhadas =sc.nextInt();
        double valorHora;
        valorHora = sc.nextDouble();

        System.out.printf("NUMBER = %d\nSALARY = U$ %.2f\n", num, (valorHora * horasTrabalhadas));
    }
}

File: Lua/bee 1008.lua
local num = tonumber(io.read("*n"))
local horas_trabalhadas = tonumber(io.read("*n"))
local valor_hora = tonumber(io.read("*n"))
print(string.format("NUMBER = %d\nSALARY = U$ %.2f", num, horas_trabalhadas*valor_hora))
